apps/charterafrica/src/payload/blocks/EmbeddedPlaylist.js

```diff
@@ -0,0 +1,193 @@
+import { select } from "payload/dist/fields/validations";
+
+import richText from "../fields/richText";
+import YouTubeSelect from "../fields/youtubeSelect/youtubeSelect";
+import {
+  BLOCK_SLUG,
+  getEmbeddedPlaylist,
+  mapPlaylistLinkToId,
+} from "../utils/embeddedPlaylist";
+
+async function validateYouTubeSelect(
+  value,
+  { data: document, hasMany, required, t }
+) {
+  let options = [];
+  const { playlistId, queryString } = getEmbeddedPlaylist(document);
+  if (playlistId) {
+    const response = await fetch(
+      `${process.env.PAYLOAD_PUBLIC_APP_URL}/api/v1/opportunities/consultation/multimedia?${queryString}`
+    );
+    const data = await response.json();
+    options =
```

Comment:
Do we still need to query options at this point?

I think it is passed as part of the object in the second parameter